toucan_connectors/google_big_query/google_big_query_connector.py

```diff
@@ -27,6 +31,24 @@
 
 _PAGE_SIZE = 50
 _MAXIMUM_RESULTS_FETCHED = 2000
+_GBQ_TIMEOUT_HTTP_REQUEST = 30  # in seconds
+
+
+class JWTNotValidExcption(GoogleUnauthorized):
+    """When the jwt-token is no longer valid (usualy from google as 401)"""
+
+
+class CustomRequestSession(requests.Session):
+    """
+    Had to create this "weird" class that wraps requests.Session
+    because the field `is_mtls` is mandatory for google...request.Session model
+    such as AuthorizedSession, and  in order to have it works properly
+    inside bigquery.Client object functions
+
+    """
+
+    # https://github.com/googleapis/google-auth-library-python/blob/v1.35.0/google/auth/transport/requests.py#L535
+    is_mtls: bool = False
```

Comment:
we currently don't ever enable mTLS on Google connectors?

Reply:
When set to True (or when thi attribute is not set), GBQ doesn't work anymore.
I didn't tried to look further on this one tho